electron/index.ts

```diff
@@ -72,6 +76,16 @@ function createWindow(connectionAtShutdown?: SerializableConnection) {
     mainWindow = null;
   });
 
+  const minimizeWindowToTray = (event: Event) => {
+    if (!mainWindow || isAppQuitting) {
+      return;
+    }
+    event.preventDefault();
```

Comment:
can you add a comment on why we call this?

Reply:
Done.